src/Components/Nav/Nav.js

```diff
@@ -60,7 +62,7 @@ const NavBox = styled.ul`
 
 const SideToggler = styled.div`
   ${({ theme }) => theme.flex('center', 'center', '')}
```

Comment:
```suggestion
  ${({ theme }) => theme.flex('center', 'center')}
```
으로 하면 더 좋을 것 같습니다~!!